pycbc/injcutter.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (C) 2013 Ian Harry
```

Comment:
Yes, that's right. If you import using __init__.py in the subpackage, you shouldn't have to edit anything that uses inject.py currently.